ten-simple-rules-dockerfiles.Rmd

```diff
@@ -86,7 +86,7 @@ To start with, we assume you have a scripted scientific workflow, i.e. you can,
 Since containers that you eventually share with others can only run open source software, tools like Mathematica and Matlab are out of scope for this example.
 A workflow that does not support scripted execution is also out of scope for reproducible research, as it does not fit well with containerization.
 Furthermore, workflows interacting with many petabytes of data and executed in high-performance computing (HPC) infrastructures are out of scope.
-Using such HPC job managers or cloud infrastructures would require a collection of "Ten Simple Rules" articles in their own right.
+Using such HPC job managers or cloud infrastructures would require a collection of "Ten Simple Rules" articles in their own right, however we would encourage the reader to look at `Singularity` [@kurtzer_singularity_2017] for this use case.
 For this article, we focus on workflows that typically run on single machine, e.g., a researchers laptop computer or a virtual server.
```

Comment:
This is a great idea to add! I think we should more cleanly separate the cloud and HPC use case - the sentence now sort of says "for HPC job managers _or_ cloud you might look at Singularity." But I think we really want to target Singularity for HPC. A suggestion:

> Using such HPC job managers or cloud infrastructures would require a collection of "Ten Simple Rules" articles in their own right. For the HPC use case, we encourage the reader to look at `Singularity` [@kurtzer_singularity_2017].

Reply:
Done!